code/modules/paperwork/records/info/he.dm

```diff
@@ -339,3 +339,14 @@
 	abno_breach_damage_type = "Red"
 	abno_breach_damage_count = "High"
 	abno_resistances = list(RED_DAMAGE = "Normal", WHITE_DAMAGE = "Weak", BLACK_DAMAGE = "Endured", PALE_DAMAGE = "Weak")//for clay dolls
+
+//Nameless Fetus
+/obj/item/paper/fluff/info/he/fetus
+	abno_type = /mob/living/simple_animal/hostile/abnormality/fetus
+	abno_code = "O-01-15"
+	abno_info = list(
+		"When the work result was Normal, the Qliphoth Counter lowered with a high probability.",
+		"When the work result was Bad, the Qliphoth Counter lowered.",
+		"When the Qliphoth Counter reached 0, Nameless Fetus began crying. Employees who heard its wails complained of a headache. Furthermore, the Qliphoth Counters of all nearby Abnormalities gradually decreased over time.",
+		"While Nameless Fetus was in a crying state, only REDACTED Work could be done.",
+		"NAME was called for by Nameless Fetus and performed REDACTED Work. They quietly walked very near Nameless Fetus. After a moment of turmoil, the crying stopped.")
```

Comment:
There is currently no REDACTED work, and it's simply based on walking into the abnormality. Please update unless this changes.

Reply:
That's what the original managerial notes said.
![image](https://user-images.githubusercontent.com/77302679/235378991-3825ef28-af27-4b60-bf92-b535c89d2401.png)

Reply:
It _is_ but we lack that at his current moment.

Reply:
You didn't do the work you just.... walked into the room and died. 
We can do the same thing, just to you know. Make it true.

Reply:
Issue is the range is 1 around it, so you walk into the room, try and use the console, and there's no "REDACTED work".